### PR TITLE
fixes (hopefully) the automorphisms

### PR DESCRIPTION
**Pull request by fieker**
2 hard examples are known to work:
  r = ray_class_field(27*maximal_order(quadratic_field(42)[1]))
  s = ray_class_field(5*maximal_order(quadratic_field(3)[1]))

### CODE REVIEW

File: src/RCF/autos.jl [hunk 1]
```diff
@@ -354,7 +356,7 @@ function find_gens(KK::KummerExt, gens_imgs::Vector{Vector{FacElem{nf_elem, Anti
         end
         found = false
         for i = 1:ngens(s)
-          if is_coprime(s.snf[i], el_in_quo[i])
+          if true || is_coprime(s.snf[i], el_in_quo[i])
```
Comment (thofma) [inline src/RCF/autos.jl:359]: Hm, this means removing the `for i = 1:...` loop?

**Comment by fieker** [comment]
Probably yes, there is room for improvement...

**Comment by thofma** [comment]
Thanks. I will add some tests tomorrow.

**Comment by fieker** [comment]
The two in the commit message plus a discriminant of a v4 Hilbert class
field?

**Pull request merged**